### PR TITLE
Validations and model specs

### PR DESCRIPTION
**Pull request by kayjeee**
Dear [Recipient's Name],

I hope this message finds you well. I am writing to request a code review for our application and to implement several essential validations. These updates are aimed at enhancing the reliability and security of our User and Post models.

For the User Model:

Name Validation: Ensure that the "Name" attribute is not blank.
PostsCounter Validation: PostsCounter must be an integer greater than or equal to zero.
For the Post Model:

Title Validation: Make sure that the "Title" attribute is not blank and does not exceed 250 characters.
CommentsCounter Validation: CommentsCounter must be an integer greater than or equal to zero.
LikesCounter Validation: LikesCounter must be an integer greater than or equal to zero.
In addition to these validations, we are also planning to add unit specifications (unit specs) for all methods and validations within our models. These unit specs will aid in ensuring the robustness and accuracy of our codebase.

Could you please allocate some time for this review and update process? It would be highly appreciated if you could provide us with feedback and suggestions to improve our code's quality.

If you require further information or clarification regarding these changes, please do not hesitate to reach out. We are committed to maintaining the highest coding standards and look forward to your guidance in this endeavor.

Thank you for your attention to this matter. Your support is instrumental in achieving our goals for a more robust and reliable application.

Best regards,

### CODE REVIEW

File: db/migrate/20230908082813_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :posts_counter, default: 0
+
+      t.timestamps
+    end
```
Comment (elisha2kyakpo1) [inline db/migrate/20230908082813_create_users.rb]: - [ ] Kindly fix the error in the migration files, you have more than one migration file with the name CreateUser.


tests

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @kayjeee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Model validations are done nicely👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @kayjeee ,

Great job so far!  

May I highlight the following: 
- [x]  Your tests are passing👏  ✔️ 
- [x] Your code is clean and well organized✔️
- [x] Well  done implementing the previous TSE's requirements✔️



## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:


 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Comment by kayjeee** [comment]
Hi @algerina 
Just growing my network and GitHub friends
just followed you, can you follow back?

**Pull request merged**